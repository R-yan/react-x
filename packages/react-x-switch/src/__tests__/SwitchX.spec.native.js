// Modules
import React from 'react';
import SwitchX from '../SwitchX';


/* eslint-disable no-undef */
describe('SwitchX.native', () => {
  set('onTintColor', () => '#000000');
  set('isDisabled', () => false);

  context('with default props', () => {
    forEach({isOn: [true, false]}, () => {
      itRenders(() => (
        <SwitchX
          isOn={isOn}
          isDisabled={isDisabled}
          onTintColor={onTintColor}
        />
      ));
    });
  });
});
