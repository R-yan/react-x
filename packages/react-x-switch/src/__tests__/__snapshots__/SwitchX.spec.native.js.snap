// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwitchX.native with default props with isOn set to false renders snapshot 1`] = `null`;

exports[`SwitchX.native with default props with isOn set to true renders snapshot 1`] = `null`;
